cookbook/form/data_transformers.rst

```diff
@@ -38,17 +152,11 @@ for converting to and from the issue number and the ``Issue`` object::
 
     class IssueToNumberTransformer implements DataTransformerInterface
     {
-        /**
-         * @var ObjectManager
-         */
-        private $om;
+        private $em;
 
-        /**
-         * @param ObjectManager $om
-         */
-        public function __construct(ObjectManager $om)
+        public function __construct(ObjectManager $em)
```

Comment:
I'm fine with that too. It should simply be consistent (having a type-hint on `ObjectManager` while the variable name indicates it is treated as an `EntityManager` implementation is a bit strange).